lib/nxt_support/middleware/sentry_error_id.rb

```diff
@@ -0,0 +1,17 @@
+module NxtSupport
+  module Middleware
+    class SentryErrorID
+      def initialize(app)
+        @app = app
+      end
+
+      def call(env)
+        status, headers, body = @app.call(env)
+        if status >= 500
+          headers["Sentry-Error-ID"] = env['sentry.error_event_id'] if env['sentry.error_event_id']
+        end
```

Comment:
I think we can combine conditions here, looks more readable. What do you think?
```suggestion
        if status >= 500 && env['sentry.error_event_id']
          headers["Sentry-Error-ID"] = env['sentry.error_event_id']
        end
```

Reply:
True